### DISCUSSION TITLE
data information

### DISCUSSION THREAD

**Discussion by wangh15** [nyumets/nyumets.github.io#4] [General]
This is a great work!!  Could you give some statistics how many pt data are openly available?   how many have follow-up?  what image sequences?   Thanks!

**Comment by RespectableGlioma** [comment]
1,429 patients with an average of 6 imaging studies per patient over an average time of 17 months for a total of 8,003 MRI studies containing the following sequences:
Expert segmentations: 2,367
T1 pre contrast: 5,542
T1 post contrast: 7,245
High-res T1 post contrast: 1,462
T2: 6,920
FLAIR: 5,322
Timepoints of clinical follow-up: 4,860
Medication updates: 81,562

**Reply by pieper** [reply]
Thanks so much for making this data available 👍 

Is there a document or page that describes the format of the data?  In particular I'm curious about the image data and expert annotations.

**Comment by RespectableGlioma** [comment]
We have an upcoming preprint that details the dataset and labelling more closely. Stay tuned!

~Eric

Eric Karl Oermann, MD
Assistant Professor of Neurosurgery, Radiology, and Data Science
NYU Langone Medical Center
New York University
Skirball, Suite 8S
530 First Avenue
New York, NY, 10016
T: 212.263.6419
E: ***@***.***
http://nyuolab.org
Associate Editor, Neurosurgery

CONFIDENTIAL: The information contained in this communication, including its attachments, may contain confidential information and is intended only for the individual(s) or entity(ies) to whom it is addressed. The information contained in this communication may also be protected by legal privilege, federal law, or other applicable law. If you are not the intended recipient of this communication, you are hereby notified that any distribution, dissemination or duplication of this communication is strictly prohibited. If you have received this communication in error, please immediately delete and destroy all copies of this message and please immediately notify us of the error by separate communication. Thank you.
